Error compiling in Expo 54

I just tried an upgrade to Expo 54. I have  react-native-fast-rsa 2.6.2 in the project.

While it compiles in SDK 53, with v54.0.7 I get the following build errors:

```
C/C++: In file included from node_modules/react-native-fast-rsa/cpp/react-native-fast-rsa.cpp:1:
C/C++: In file included from node_modules/react-native-fast-rsa/cpp/react-native-fast-rsa.h:3:
C/C++: In file included from node_modules/react-native/ReactCommon/jsi/jsi/jsilib.h:10:
C/C++: node_modules/react-native/ReactCommon/jsi/jsi/jsi.h:79:5: error: no matching function for call to 'snprintf'
C/C++:    79 |     std::snprintf(
C/C++:       |     ^~~~~~~~~~~~~
C/C++: /Users/jamesholmes/Library/Android/sdk/ndk/27.1.12297006/toolchains/llvm/prebuilt/darwin-x86_64/sysroot/usr/include/stdio.h:286:5: note: candidate function not viable: 1st argument ('const value_type *' (aka 'const char *')) would lose const qualifier
C/C++:   286 | int snprintf(char* __BIONIC_COMPLICATED_NULLNESS __buf, size_t __size, const char* _Nonnull __fmt, ...) __printflike(3, 4);
C/C++:       |     ^        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
C/C++: /Users/jamesholmes/Library/Android/sdk/ndk/27.1.12297006/toolchains/llvm/prebuilt/darwin-x86_64/sysroot/usr/include/bits/fortify/stdio.h:77:5: note: candidate function not viable: 1st argument ('const value_type *' (aka 'const char *')) would lose const qualifier
C/C++:    77 | int snprintf(char* const __BIONIC_COMPLICATED_NULLNESS __pass_object_size dest, size_t size, const char* _Nonnull format, ...)
C/C++:       |     ^        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
C/C++: 1 error generated.
```

--- a/android/CMakeLists.txt
+++ b/android/CMakeLists.txt
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 3.9.0)
 
 set (CMAKE_VERBOSE_MAKEFILE ON)
-set (CMAKE_CXX_STANDARD 14)
+set (CMAKE_CXX_STANDARD 17)
 set (BUILD_DIR ${CMAKE_SOURCE_DIR}/build)
 
 add_library(rsa_bridge SHARED IMPORTED)

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -69,7 +69,7 @@ android {
       cmake {
         cppFlags "-O2 -frtti -fexceptions -Wall -Wno-unused-variable -fstack-protector-all"
         abiFilters "x86", "x86_64", "armeabi-v7a", "arm64-v8a"
-        arguments "-DNODE_MODULES_DIR=${rootDir}/../node_modules"
+        arguments "-DNODE_MODULES_DIR=${rootDir}/../node_modules", "-DANDROID_SUPPORT_FLEXIBLE_PAGE_SIZES=ON"
       }
     }
   }

